gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -3132,6 +3132,14 @@ GbGradeTable.localizeNumber = function(number) {
     return '' + number;
 };
 
+GbGradeTable.localizedStringToNumber = function(localizedString) {
```

Comment:
can you add a comment about how this function works?

Reply:
Hi @ottenhoff , I've improved and added some comments to the function